wom/src/main/scala/wdl4s/wom/graph/Graph.scala

```diff
@@ -8,6 +8,8 @@ import wdl4s.wom.graph.GraphNodePort.InputPort
   * A sealed set of graph nodes.
   */
 final case class Graph private(nodes: Set[GraphNode]) {
+  // FIXME: this actually seems pretty WDL specific, and even then specific to the case of a WDL that doesn't have
+  // an explicit outputs block.  I'd remove this but the WDL/WOM test relies on it.
```

Comment:
I think it should probably move somewhere in the `wdl4s.wdl` package (or maybe WDL already has a function for this and we should use that instead).

FWIW I think it's more likely to get fixed with a ticket than a code FIXME, but that's not a strong opinion either way